apps/community/views.py

```diff
@@ -16,40 +15,17 @@ class FilterListView(ListView):
     template_name = 'community/index.html'
     context_object_name = "filtered"
     paginate_by = 5
-    allow_empty = True
 
     def get_context_data(self, **kwargs):
         data = super(FilterListView, self).get_context_data(**kwargs)
-        data['filter'] = {}
-
-        username = self.request.GET.get('username', False)
-        
-        if username:
-            data['filter']['username'] = username
-
-        company = self.request.GET.get('company', False)
-        if company:
-            data['filter']['company'] = company
-        
-        email = self.request.GET.get('email', False)
-        if email:
-            data['filter']['email'] = email
-        
-        projectname = self.request.GET.get('projectname', False)
-        if projectname:
-            data['filter']['projectname'] = projectname
-        
-        category = self.request.GET.get('category', False)
-        if category:
-            data['filter']['category'] = category
-        
-        manufacturer = self.request.GET.get('manufacturer', False)
-        if manufacturer:
-            data['filter']['manufacturer'] = manufacturer
-
-        qdict = QueryDict('', mutable=True)
-        qdict.update(data['filter'])
-        data['filter_url'] = qdict.urlencode()
+        data['filter'] = {
```

Comment:
@12jr Hier hat sich noch was altes eingeschlichen. Bitte den Master nehmen!

Reply:
ich habe von Maren und huy gepullt, da kommt es wahrscheinlich her... aber projectsH&M hat schon wieder das richtige oder?

Reply:
habe das aus dem Master in https://github.com/IT-PM-OpenAdaptronik/Webapp/commit/39d3a6544f3c612aef31c5a94433d569690a2d73 reinkopiert